libs/components/package.json

```
@@ -40,6 +40,7 @@
     "./nav-disclosure": "./nav-disclosure",
     "./nav-item": "./nav-item",
     "./note": "./note",
+    "./listbox-option": "./listbox-option",
```

https://github.com/microsoft/fast/blob/87b817259c504fbcae23bf12a6039fc646b2d7a6/packages/web-components/fast-foundation/src/listbox-option/README.md?plain=1#L9

this only affects the class itself and not necessarily the folder structure.
we either export it as option or even rename the path and filenames to plainly _**option**_
```suggestion
    "./option": "./listbox-option",
```